Libraries/react-native/react-native.js

```diff
@@ -10,14 +10,18 @@
  */
 'use strict';
 
+var React = require('React');
+
 // Export React, plus some native additions.
 //
 // The use of Object.create/assign is to work around a Flow bug (#6560135).
 // Once that is fixed, change this back to
 //
-//   var ReactNative = {...require('React'), /* additions */}
+//   var ReactNative = {...React, /* additions */}
 //
-var ReactNative = Object.assign(Object.create(require('React')), {
+var ReactNative = Object.assign(Object.create(React), {
+  React,
```

Comment:
https://github.com/facebook/react/pull/3221

Personally this strikes me as a little weird and I don't want it.